libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -264,7 +264,15 @@ const AP_Param::GroupInfo AP_TECS::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("PTCH_FF_K", 30, AP_TECS, _pitch_ff_k, 0.0),
 
-    // 31 previously used by TECS_LAND_PTRIM
+    // 31 previously used by AP_Int8 TECS_LAND_PTRIM which was removed in November 2022
+
+    // @Param: THR_ERATE
```

Comment:
Do we need a new param? Can't we use the existing plane throttle slew limit?

Reply:
It needs to be slower based on flight testing where the tilting rotors were energising the front wing in a tandem wing configuration.